src/player.cpp

```diff
@@ -6042,6 +6044,10 @@ void player::suffer()
         body_part bp = random_body_part(true);
         add_effect( effect_formication, 10_minutes, bp );
     }
+    if (has_bionic ( bio_glowy ) && one_in(500) !has_effect( effect_glowing )) {
+        add_msg_if_player(m_bad, _("Your malfunctioning bionic starts to glow!"));
```

Comment:
```suggestion
    if( has_bionic( bio_glowy ) && !has_effect( effect_glowing ) && one_in( 500 ) ) {
        add_msg_if_player( m_bad, _( "Your malfunctioning bionic starts to glow!" ) );
```

Did your original version even compile?  Please compile and test your changes before pushing them.